src/components/MailingLists/MailingLists.tsx

```diff
@@ -0,0 +1,45 @@
+import { TabsMenu, TabsMenuItem, Text } from '@taskany/bricks';
+import { useEffect, useState } from 'react';
+
+import { mailingSettingType } from '../../modules/userTypes';
+import { MailingList } from '../MailingList/MailingList';
+import { AdminPanelLayout } from '../AdminPanelLayout/AdminPanelLayout';
+import { trpc } from '../../trpc/trpcClient';
+import { getOrgUnitTitle } from '../../utils/organizationUnit';
+
+import { tr } from './MailingLists.i18n';
+
+export const MailingLists = () => {
+    const organizationUnitQuery = trpc.organizationUnit.getList.useQuery({ search: '' });
+
+    const [orgUntId, setOrgUnitId] = useState<string | undefined>(undefined);
+    useEffect(() => {
```

Comment:
What this effect is doing?

Reply:
automatically set first organization id to selected tab